ngraph_bridge/ngraph_builder.cc

```diff
@@ -5232,7 +5242,7 @@ const static std::map<
       {"TanhGrad", TranslateTanhGradOp}, {"Tile", TranslateTileOp},
       {"TopKV2", TranslateTopKV2Op}, {"Transpose", TranslateTransposeOp},
       {"UnsortedSegmentSum", TranslateUnsortedSegmentSumOp},
-      {"Unpack", TranslateUnpackOp}, {
+      {"Unpack", TranslateUnpackOp}, {"LogicalXor", TranslateXorOp}, {
```

Comment:
Are we adding support for Xor or LogicalXor? Both? Seeing different TF op types in builder vs mark for clustering